### ISSUE TITLE
Add Ceritinib to Hetionet?

### ISSUE THREAD

**Issue by sfrally** [dhimmel/integrate#17]
Thanks for the work setting up Hetionet!

Is it possible to add the cancer drug Ceritinib (https://www.drugbank.ca/drugs/DB09063)?

A similar compound Crizotinib (https://www.drugbank.ca/drugs/DB08865) is in the database.

They both target the same protein for cancer treatment and it would be great to see how they differentiate within Hetionet.

**Comment by dhimmel** [comment]
I am guessing Ceritinib is missing from Hetionet because it was added to DrugBank shortly after the version 4.2 release which [we used](https://doi.org/10.15363/thinklab.d40). I see on the DrugBank entry it says "Drug created on May 11, 2015".

You bring up a good point: we could make Hetionet more useful by updating it and expanding the coverage of diseases and compounds. Unfortunately, we don't have any releases planned in the immediate future. Although it is possible we decide to release Hetionet 2.0 at some point, at which point Ceritinib would be highly likely to be included.

The architecture of the resource makes it so we're likely to do major updates of all resources rather than adding a single node on an _ad hoc_ basis. I will try to remember to comment here if I see any other hetnets that update Hetionet and may contain Ceritinib.

Finally, if there are specific edge types you care about, I could point you to the right repositories to allow you to compute the edges.
